.github/workflows/stale_issue_pr.yaml

```diff
@@ -0,0 +1,27 @@
+name: 'Stale issue & PR handler'
```

Comment:
Do we need stale issue & PR handler activated in release branch as well ? Is there a way to restrict this workflow to master branch only ?

Reply:
Synced up offline, workflows are only driven from the files in master branch.